woocommerce/woocommerce-gutenberg-products-block
# Add Cart Totals to cart i2

Part of #4821
This adds `Cart Totals` to the correct place.
It also moves a useEffect to the correct place.


### How to test
- Add cart i2.
- Go to frontend.
- You should see `Cart Totals`
- The use effect was missed as part of the refactor, you can try triggering by entering a wrong postcode in shipping calculator.
